src/util/Constants.js

```diff
@@ -58,13 +58,15 @@ exports.DefaultOptions = {
   /**
    * WebSocket options (these are left as snake_case to match the API)
    * @typedef {Object} WebsocketOptions
-   * @property {number} [large_threshold=250] Number of members in a guild to be considered large
+   * @property {number} [large_threshold=50] Number of members in a guild to be considered large
    * @property {boolean} [compress=false] Whether to compress data sent on the connection
+   * @property {boolean} [guild_subscriptions=true] enables dispatching of guild subscription events (presence and typing events)
```

Comment:
```suggestion
   * @property {boolean} [guild_subscriptions=true] enables dispatching of guild subscription events
   * (presence and typing events)
```
line length was over 120, breaking the max-line-length rule